.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: PYBASEBALL-BUG-
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
```

Comment:
IMO I think the first three forms here are going to most often yield the same thing 3 times over and over: "I expected this to happen when I did this." I'd prefer one simpler form, something like 

```
***Describe the bug and how to produce it***
```